README.md

```diff
@@ -80,6 +80,20 @@ $ docker-compose run --rm api rake db:create db:schema:load linksf:import
 $ docker-compose run -e STAGING_DB_PASSWORD=<...> --rm api rake db:setup db:import_staging
 ```
 
+### Running tools from the terminal
+Due to the DB and API being contained inside Docker containers, running ruby/rails specific tools without having to ssh into the containers involves specific steps
+#### Running the rails console from the terminal
+```sh
+# Start both the DB and API
+$ cd PATH/TO/askdarcel-api
+$ docker-compose run --rm api rails console
```

Comment:
While I think it's useful to have this here, I'm not sure whether it belongs under a section like this.

The general rule of thumb is for any command that you would normally native run, just prefix it with `docker-compose run --rm api`.

Also, I wasn't exactly sure what you meant by "ssh into the containers". We don't literally SSH into the containers with `ssh`, but the closest thing to that is executing a command within the container while attaching your terminal's stdin/stdout. But that is exactly what you are doing here with this command, which seems to conflict with the sentence you wrote at the beginning of the section.

Reply:
Good points. I have updated the wording to be, hopefully, correct and removed the contradictions.

I renamed the section to be `Developer Tools`; do you think that is a better section have.